src/get-started/java-binding.md

```diff
@@ -1102,7 +1102,8 @@ aliases are used in other parts of the configuration for readability and easier
 refactoring. Java artifacts also need the name of the JAR file in the
 `spec: artifact_filename` field of the artifacts directory. The present example
 shows how to add the Java `cryptocurrency-demo` service, and two Rust services —
-the `timestamping` and `time` oracle services.
+the `timestamping` and `time` oracle services. Please note, that Rust services
+are pre-deployed in the genesis block, so we don't use `action` field for them.
```

Comment:
We do for services, therefore, I'd clarify that we do not _deploy_ them:

... we don't specify `action: deploy` for them.

... we do not deploy their artifacts (= specify `action: deploy`).